### DISCUSSION TITLE
Are submissions closed?

### DISCUSSION THREAD

**Discussion by khushii-ag** [cyberbotics/wrestling#89] [General]
Please confirm if the submissions are closed and no further submissions will be accepted.

**Comment by omichel** [comment]
Yes, they are closed. See my messages [here](https://github.com/cyberbotics/wrestling/discussions/63#discussioncomment-5976165).

**Reply by JustLuckyRobotics** [reply]
Does this also mean if we make a small tweak to something now after viewing our last replay the commit wont be taken? obviously it wont run anymore but when will the snapshot be taken?

**Reply by omichel** [reply]
No please don't make any changes now to your repositories or do it at your own risk as I am currently downloading all the repos...

**Reply by omichel** [reply]
The snapshot are being taken now...

**Reply by JustLuckyRobotics** [reply]
We committed last at 14:25 with the hope it might, very minor value tweak tested on local, nothing dangerous to our chances. If its taken great, if not so be it.

**Reply by omichel** [reply]
You are lucky. I checked and it seems it's taken. But now the download of the snapshots is over. So no further change is possible.

**Reply by JustLuckyRobotics** [reply]
why do you think my name is what it is? =]
